litex/tools/litex_json2dts_linux.py

```diff
@@ -328,7 +328,7 @@ def generate_dts(d, initrd_start=None, initrd_size=None, initrd=None, root_devic
 
     # Interrupt Controller -------------------------------------------------------------------------
 
-    if (cpu_arch == "riscv") and ("rocket" in cpu_name):
+    if (cpu_arch == "riscv") and ("rocket" in cpu_name or "vexiiriscv" in cpu_name):
```

Comment:
Why not using:
```python
if (cpu_arch == "riscv") and (cpu_name in ["rocket",  vexiiriscv"]):
```

Reply:
i was afraid of breaking some weird stuff as : 
"rocket" in cpu_name isn't equivalent to cpu_name in ["rocket"]
As there may eventualy be a "rocket_on the moon" as cpu_name ?

So i prefered not disrupting the old behaviour. As they could have done it with cpu_name == "rocket" but didn't XD

Reply:
make sense but `cpu_name` is used with `cpu_architecture` so if there is a "rocket_somewhere" a keyError will be produces. :)

Reply:
I pushed as you proposed just now ^^